test/TestLocalContent.py

```diff
@@ -0,0 +1,42 @@
+"""Test ability to import playbooks."""
+from ansiblelint.runner import Runner
+
+
+def test_local_collection(default_rules_collection):
+    """Assures local collections are found."""
+    playbook_path = 'test/local-content/test-collection.yml'
+    runner = Runner(default_rules_collection, playbook_path, [], [], [])
+    results = runner.run()
+
+    assert len(runner.playbooks) == 1
+    assert len(results) == 0
+
+
+def test_roles_local_content(default_rules_collection):
+    """Assures local content in roles is found."""
+    playbook_path = 'test/local-content/test-roles-success.yml'
+    runner = Runner(default_rules_collection, playbook_path, [], [], [])
+    results = runner.run()
+
+    assert len(runner.playbooks) == 1
+    assert len(results) == 0
+
+
+def test_roles_local_content_failure(default_rules_collection):
+    """Assures local content in roles is found, even if Ansible itself has trouble."""
+    playbook_path = 'test/local-content/test-roles-failure.yml'
+    runner = Runner(default_rules_collection, playbook_path, [], [], [])
+    results = runner.run()
+
+    assert len(runner.playbooks) == 1
+    assert len(results) == 0
+
+
+def test_roles_local_content_failure_complete(default_rules_collection):
+    """Role with local content that is not found."""
+    playbook_path = 'test/local-content/test-roles-failure-complete.yml'
+    runner = Runner(default_rules_collection, playbook_path, [], [], [])
+    results = runner.run()
```

Comment:
This run should fail, but since I cannot run the tests, and CI doesn't run them either, I have no idea how to write this correctly to catch the error.